crossterm_terminal/src/terminal/ansi_terminal.rs

```diff
@@ -85,20 +85,19 @@ mod tests {
     use super::{AnsiTerminal, ITerminal};
 
     /* ======================== ANSI =========================== */
-    // TODO - Test is disabled, because it's failing on Travis CI
     #[test]
-    #[ignore]
```

Comment:
This doesn't work on TCI, should be disabled. See the comment above (in the code).